config/jwt/private.pem

```diff
@@ -0,0 +1,54 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This certs are for development only. Need to be overwritten for production.

Reply:
I suggest a Make command to generate these certs.

Like this:
```
// Makefile
generate-jwt-certs:
	docker-compose exec php sh -lc 'openssl genrsa -out config/jwt/private.pem -aes256 4096'
	docker-compose exec php sh -lc 'openssl rsa -pubout -in config/jwt/private.pem -out config/jwt/public.pem'
```

Reply:
No openssl in the container. Having this one for dev and ci it's enough as the certs for prod use to be quite customized.

Don't see any good reason to add this. But if you give me one I'll do it ;)